### DISCUSSION TITLE
[FeatureRequest] AlertList: Support template variables in filter tags

### DISCUSSION THREAD

**Discussion by lmuhlha** [grafana/grafana#37023] [Dashboards, Panels & Visualizations]
**What would you like to be added**:

I'd like to be able to use a variable in the tag filter for the alert list ~~and dashboard list~~ visualization. Currently this does not work. 

For example, trying to use https://grafana.test.net/d/test/test?orgId=1&var-test=test

![Screen Shot 2020-04-02 at 11 15 30 AM](https://user-images.githubusercontent.com/16561599/78266089-4ef7b700-74d3-11ea-97f6-eb93a733d960.png)

**Why is this needed**:
Using variables and plugging them in via the url works for other visualization types. Would expect it to work in these as well.

**Comment by hugohaggmark** [comment]
This has already been partially reported in #29028 so I'm going to clean up this issue's title and description to only cover alert list.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
